x-pack/plugins/security_solution/public/explore/network/components/embeddables/embedded_map.tsx

```diff
@@ -123,47 +123,48 @@ export const EmbeddedMapComponent = ({
   const isFieldInIndexPattern = useIsFieldInIndexPattern();
 
   const [mapDataViews, setMapDataViews] = useState<SourcererDataView[]>([]);
+  const [availableDataViews, setAvailableDataViews] = useState<SourcererDataView[]>([]);
 
-  const availableDataViews = useMemo(() => {
+  const isMountedRef = useRef(true);
+  useEffect(() => {
+    return () => {
+      isMountedRef.current = false;
```

Comment:
this was the main problem, need to isolate the `isMountedRef` var (prev `canceled`) to an empty dependency useEffect.